src/views/PlayingNow.js

```diff
@@ -1,7 +1,249 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Cover from '../components/Cover';
+import MediaControls from '../components/MediaControls';
+import GridContainer from '../components/Grid/GridContainer';
+import GridItem from '../components/Grid/GridItem';
+import AlbumsList from '../components/AlbumsList';
+import DeviceSelector from '../components/DeviceSelector';
+import VolumeControl from '../components/VolumeControl';
+import PropTypes from 'prop-types';
+import {
+  getCurrentlyPlaying,
+  nextSong,
+  previousSong,
+  getPlayer,
+  setVolume,
+  getDevices,
+  pause,
+  play,
+  setRepeat,
+  toggleShuffle
+} from '../actions/playerActions';
 
-const PlayingNow = () => {
-  return <h1>Playing Now</h1>;
+const currentPlayback = {
```

Comment:
Why is this hardcoded?

Reply:
That constant is used on the defaultProps